config-templates/ibm/ibm-csi-block/1.4.0/README.md

```diff
@@ -0,0 +1,101 @@
+# IBM block storage Container Storage Interface driver
+
+IBM block storage CSI driver is based on an open-source IBM project, included as a part of IBM Storage orchestration for containers. IBM Storage orchestration for containers enables enterprises to implement a modern container-driven hybrid multicloud environment that can reduce IT costs and enhance business agility, while continuing to derive value from existing systems.
+
+For full release notes, compatiblity, installation, and user information, see the IBM block storage CSI driver documentation: https://www.ibm.com/support/knowledgecenter/SSRQ8T_1.4.0/
+
+## Prerequisites
+
+For full prerequisite instructions, see the following documentation, referred to in the introductory paragraph. 

```

Comment:
```suggestion
- Review the [compatibility and requirements documentation](https://www.ibm.com/support/knowledgecenter/SSRQ8T_1.4.0/csi_ug_requirements.html). 

```

Reply:
Which steps do they need to complete on this page or others? It would be good to have a list of prereqs with links like 
```
- Review the compatibility and requirements
- [Install the required packages](link)
- etc
```

Reply:
done
I think the documentation is good enough, if we will right it again here we could miss or change things that will cause issues.
they need to go over all of it.